NEWS.rst

```diff
@@ -2,6 +2,46 @@
 Release Notes
 =============
 
+----------------------------
+pkgcheck 0.10.18 (2022-11-09)
+----------------------------
+
+Special thanks is given to Sam James, for continues support during all
+development, bringing ideas, testing and improving, and especially proofreading
+and improving all docs, texts and help messages. Every release is better thanks
+to him.
+
+- Network checks: fix wrong attributes shown when same URL is checked (#403,
+  Arthur Zamarin)
+
+- BetterCompressionCheck: new check for suggesting better compression URI for
+  distfile fetch (#483, Arthur Zamarin)
+
+- ExcessiveLineLength: report lines longer than 120 characters (with multiple
+  exception rules) (#480, Arthur Zamarin)
+
+- MissingRemoteIdCheck: new check for suggesting missing remote ids, inferred
```

Comment:
I think:
```suggestion
- MissingRemoteIdCheck: new check for suggesting missing remote-ids, inferred
```

This is kind of a nit though and what you did is technically fine, it's just that in a Gentoo context, they're _also_ "remote-id" as that's the name in XML.